tests/tracer/test_monkey.py

```diff
@@ -31,6 +31,13 @@ def test_patch_all_env_override_manual_patch(self):
         monkey.patch(sqlite3=True)
         assert "sqlite3" in monkey._PATCHED_MODULES
 
+    @run_in_subprocess(env_overrides=dict())
+    def test_patch_raise_exception_manual_patch(self):
+        # Manual patching should not be affected by the environment variable override.
+        with self.assertRaises(monkey.ModuleNotFoundException):
```

Comment:
can we assert the exception message?

Reply:
or if we can make sure we have tests that assert the log message we end up showing, should be good enough?